src/components/forms/RampForm/index.tsx

```diff
@@ -67,6 +67,9 @@ export default function RampForm({ noContainer = false }: Readonly<RampFormProps
       if ((err as AxiosError).response?.status === 403) {
         setFormError(t('errors.forbidden'));
         return;
+      } else if ((err as AxiosError).response?.status === 406) {
```

Comment:
taking into account the conditions we current have, we could refactor this like:
```
const getErrorMessage = (status?: number) =>  {
  switch (status) {
    case 403:
        return 'errors.forbidden'
    case 406:
      return  'errors.accountNames'
    default:
      return 'errors.recipient'
  }
}

// on the catch
} catch (err) {
  setFormError(t(getErrorMessage((err as AxiosError).response?.status)))
}
```